rclcpp/include/rclcpp/logger.hpp

```diff
@@ -77,6 +78,14 @@ RCLCPP_PUBLIC
 Logger
 get_node_logger(const rcl_node_t * node);
 
+// TODO(ahcorde): Remove deprecated class on the next release.
```

Comment:
last-minute minor nitpick, feel free to ignore: it would be a lot easier on future-us to mention the name of the release here, e.g., "in Rolling after Kilted"